sources/packages/api/src/esdc-integration/e-cert-full-time-integration/e-cert-files/e-cert-file-record.ts

```diff
@@ -0,0 +1,229 @@
+   */
+  totalYearsOfStudy: number;
+  /**
+   * Date that the school confirmed enrolment.
+   */
+  enrollmentConfirmationDate: Date;
+  /**
+   * Student date of birth.
+   */
+  dateOfBirth: Date;
+  /**
+   * Student last name.
+   */
+  lastName: string;
+  /**
+   * Student first name.
+   */
+  firstName: string;
+  /**
+   * Student address.
+   */
+  addressLine1: string;
+  /**
+   * Student address
+   */
+  addressLine2: string;
+  /**
+   * Student city address.
+   */
+  city: string;
+  /**
+   * Student country Name.
+   */
+  countryName: string;
+  /**
+   * Student email address.
+   */
+  emailAddress: string;
+  /**
+   * Student gender M=Male F=Female.
+   */
+  gender: string;
+  /**
+   * Borrower marital status S= Single, M = Married, O = Other.
+   */
+  maritalStatus: string;
+  /**
+   * Student number (at the Educational Institution).
+   */
+  studentNumber: string;
+  /**
+   * Total of all grant awards on this Entitlement Record.
+   */
+  totalGrantAmount: number;
+  /**
+   * Federal grants list (maximum 10) where 6 to 10 are not expecting data.
+   * ! All values must be rounded, only integers, no decimals.
+   */
+  grantAwards: Award[];
+
+  public getFixedFormat(): string {
+    const record = new StringBuilder();
+    record.append(this.recordType);
+    record.append(this.sin, 9);
+    record.appendWithStartFiller(this.applicationNumber, 19, NUMBER_FILLER);
+    record.appendWithStartFiller(this.documentNumber, 8, NUMBER_FILLER);
+    record.appendDate(this.disbursementDate, DATE_FORMAT);
+    record.appendDate(this.documentProducedDate, DATE_FORMAT);
+    record.appendDate(this.negotiatedExpiryDate, DATE_FORMAT);
+    record.appendWithStartFiller(this.disbursementAmount, 6, NUMBER_FILLER);
+    record.appendWithStartFiller(this.studentAmount, 8, NUMBER_FILLER);
+    record.appendWithStartFiller(this.schoolAmount, 8, NUMBER_FILLER);
+    record.appendWithStartFiller(this.cslAwardAmount, 6, NUMBER_FILLER);
+    record.appendWithStartFiller(this.bcslAwardAmount, 6, NUMBER_FILLER);
+    record.appendDate(this.educationalStartDate, DATE_FORMAT);
+    record.appendDate(this.educationalEndDate, DATE_FORMAT);
+    record.appendWithEndFiller(this.federalInstitutionCode, 4, SPACE_FILLER);
+    record.appendWithStartFiller(this.weeksOfStudy, 2, NUMBER_FILLER);
+    record.appendWithStartFiller(this.fieldOfStudy, 2, NUMBER_FILLER);
+    record.append(this.yearOfStudy.toString(), 1);
+    record.append(this.totalYearsOfStudy.toString(), 1);
+    record.repeatAppend(SPACE_FILLER, DATE_FORMAT.length); // Cancel Date, optional, not provided.
+    record.append("F"); // 'F' for full-time. Part time is done by another integration to another system.
+    record.repeatAppend(SPACE_FILLER, 2); // Provincial field of study code, optional, not provided.
+    record.appendDate(this.enrollmentConfirmationDate, DATE_FORMAT);
+    record.append("E"); // Indicates if e-cert or paper certificate E=E-cert; P=Paper.
```

Comment:
so when comes the `P=Paper` scenario/condition?

Reply:
My understanding so far is "never" 😄 
@abschwenker, do you have more context on this?

Reply:
Everything is digital now afaik, this existed for a point of time to support transition. So "Never" is correct